web/modules/custom/itk_siteimprove/js/itk-siteimprove.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * @file
+ * Adds siteimprove to the current page.
+ */
+
+(function ($, Drupal) {
+  'use strict';
+
+  Drupal.behaviors.cookieMessageBehavior = {
+    attach: function (context, settings) {
+      var key = drupalSettings.itk_siteimprove.key;
+
+      if (key) {
+        (function () {
+          var sz = document.createElement('script');
+          sz.type = 'text/javascript';
+          sz.async = true;
+          sz.src = '//ssl.siteimprove.com/js/siteanalyze_' + key + '.js';
```

Comment:
They don't have any version information in there scripts?

Reply:
No